srcpkgs/zam-plugins/files/zam-plugins.directory

```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Name=zam-plugins
+Icon=zam-plugins
+Type=Directory
+Keywords=audio;sound;jackd;zam-plugins;
```

Comment:
Don't they provide this file?

Reply:
They don't provide it. I've taken this file from Arch Linux package, and credited it's maintainer in `template`.

Reply:
I strongly disagree with this policy and consider it upstream's responsibility to have such files. Otherwise all distros have to carry the same mess; instead, the files should be upstreamed (and ideally installed automatically).

Reply:
I've removed them already